src/macros.rs

```diff
@@ -176,7 +176,8 @@ macro_rules! wrap_pymodule {
         &|py| {
             use $module as wrapped_pymodule;
             wrapped_pymodule::_PYO3_DEF
-                .make_module(py)
+                // corrected in add_wrapped later based on the parent module's settings
```

Comment:
```suggestion
```

I think this comment is a remnant of an earlier approach. No need to explain why we default to not supporting free-threaded.